test/azure/version-tolerant/Expected/AcceptanceTests/StorageManagementClientVersionTolerant/storageversiontolerant/aio/operations/_operations.py

```diff
@@ -990,7 +989,6 @@ def prepare_request(next_link=None):
                 request = build_storage_accounts_list_by_resource_group_request(
                     resource_group_name=resource_group_name,
                     subscription_id=self._config.subscription_id,
-                    api_version=api_version,
```

Comment:
@johanste are we good with removing `api_version` formatting right now? We can add it back (either in core or in generation) once archboard has decided